src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IDelegateCreationExpression.vb

```diff
@@ -0,0 +1,966 @@
+' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.Test.Utilities
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Roslyn.Test.Utilities
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.Semantics
+
+    Partial Public Class IOperationTests
+        Inherits SemanticModelTestBase
```

Comment:
>SemanticModelTestBase [](start = 17, length = 21)

Also New DelegateType with two or more arguments.

Reply:
Added.

---
In reply to: [138755797](https://github.com/dotnet/roslyn/pull/22056#discussion_r138755797) [](ancestors = 138755797)